pkg/agent/cniserver/pod_configuration.go

```diff
@@ -196,6 +196,13 @@ func ParseOVSPortInterfaceConfig(portData *ovsconfig.OVSPortData, portConfig *in
 	return interfaceConfig
 }
 
+func (pc *podConfigurator) configureInterfacesMTU(containerNetNS string, containerIFDev string, mtuDeduction int) error {
```

Comment:
No need to add this func. You can directly call `s.podConfigurator.ifConfigurator.configureContainerMTU()` in CNIServer.

Reply:
Done